components/TextFields/src/MDCTextInputControllerLegacyDefault.m

```diff
@@ -542,12 +537,19 @@ - (void)movePlaceholderToUp:(BOOL)isToUp {
       }];
 }
 
+- (BOOL)isPlaceholderUp {
+  return self.placeholderAnimationConstraints.count > 0 &&
+  !CGAffineTransformEqualToTransform(self.textInput.placeholderLabel.transform,
```

Comment:
Nit/Indentation: The second row should be indented 4 spaces.

Reply:
Done.